blank-canvas-blocks/sass/elements/_links.scss

```diff
@@ -13,7 +13,7 @@ a:not(.wp-block-button__link) {
 }
 
 // Links that are in the content
-.entry-content a:not(.wp-block-button__link) {
+.entry-content a:not(.wp-block-button__link):not(.wp-block-file__button) {
```

Comment:
Regarding the nots see some discussion here: https://github.com/Automattic/themes/pull/3478#discussion_r596028025

Do you think we can overcome these styles in the `.wp-bloc-file__button` class instead of proving a :not() here?